### DISCUSSION TITLE
Duvidas em uma linha de código

### DISCUSSION THREAD

**Discussion by Paulo23k** [Paulo23k/OOP-DEV#3] [Q&A] [answered]
echo implode(", ", array_map(fn($x) => preg_replace('/\d/', '#', $x), array_filter(['user1', 'admin123', 'guest!@#', 'admin456'], fn($x) => preg_match('/admin/', $x))));

**Comment by Libann23k** [comment, marked as answer]
O código filtra elementos de um array contendo a palavra "admin", substitui os números por "#" nas strings filtradas e os une em uma única string separada por vírgulas.
